ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.less

```diff
@@ -62,14 +59,18 @@
 
 @{customElementNamespace}filter.is-location {
   > .filter-input {
-    margin-top: @minimumSpacing;
     display: block;
+    padding: @minimumSpacing 1.5rem 0px @minimumSpacing + 0.75 *
```

Comment:
❓ Some of the numbers in this padding seem a little arbitrary (1.5rem, and the calculated last one) do these work across spacing settings in the theme?

Reply:
So 1.5rem is arbitrary, I just found that that's the padding that looked good with all the spacing settings. And the `@minimumSpacing + 0.75*@minimumButtonSize + @minimumButtonSize` is calculated to match up with the start of the filter dropdown no matter what spacing settings you're using.